api/v1alpha1/agentconfig_types.go

```diff
@@ -0,0 +1,45 @@
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type AgentConfigurationSpec struct {
+	ClusterPingInterval               *string `json:"clusterPingInterval,omitempty"`
+	RuntimeServicesPingInterval       *string `json:"runtimeServicesPingInterval,omitempty"`
```

Comment:
Lets name this `compatibilityUploadInterval` (users probably don't know the runtime service terminology)